synapse/tests/test_lib_cache.py

```diff
@@ -58,3 +59,35 @@ def woot(x):
         woot.cache.put((40,), 99)
         self.eq(99, woot(40))
         self.eq(misses, 2)
+
+    async def test_tag_globs(self):
+
+        base = await s_base.Base.anit()
+
+        glob = s_cache.TagGlobs()
+
+        glob.add('zip.*', 1)
+        glob.add('zip.*si', 11)
+        glob.add('foo.*.baz', 2)
+        glob.add('a.*.*.c', 3, base=base)
+
+        vals = tuple(sorted(v[1] for v in glob.get('foo.bar.baz')))
+        self.eq(vals, (2,))
+
+        vals = tuple(sorted(v[1] for v in glob.get('zip.visi')))
+        self.eq(vals, (1, 11))
```

Comment:
In 00x we had the tag glob convention for storm where `foo.*.bar` would match a single wildcard tag level, and `foo.**.bar` would match any string between foo and bar.  

The behavior of the regex defined for the TagGobs differs; is this the same style of globbing we would want to introduce into storm down the road?

Reference:  https://github.com/vertexproject/synapse/blob/v0.0.56/synapse/lib/storm.py#L873

Reply:
We may want to, but if so, we'd probably want it to be optional and not enabled for tag globbing.  the ** style wild carding would potentially cause triggers to fire multiple times for a tag add that was multi-level.